api/bases/test.openstack.org_tempests.yaml

```diff
@@ -458,6 +480,28 @@ spec:
                       description: An URL of a tempest container image that should
                         be used for the execution of tempest tests.
                       type: string
+                    extraMounts:
```

Comment:
I suggest we remove the `extraMounts` parameter from the workflow section. The parameters outside of the `tempestRun` and `tempestconfRun` sections are meant to be global parameters that should not be overridden by workflow configuration. 

Also I wonder whether `extraMounts` is the correct name. What about `extraConfigMapMounts`?

Reply:
ok